scripts/ssh_session.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# setting the locale, some users have issues with different locales, this forces the correct one
+export LC_ALL=en_US.UTF-8
+
+show_ssh_session_port=$1
+
+parse_ssh_port() {
+  # Get port from connection
+  local port=$(echo $1|grep -Eo '\-p\s*([0-9]+)'|sed 's/-p\s*//')
+
+  if [ -z $port ]; then
+    local port=22
+  fi
+
+  echo $port
+}
+
+get_ssh_user() {
```

Comment:
Could this function be cleaned up a bit? Looks like a lot of duplicate code.

Reply:
Function has been cleaned up and documented, thank you for pointing it out